src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -24,20 +27,18 @@ static Https_ClientCredentialTypeTests()
 #if FULLXUNIT_NOTSUPPORTED
     [Fact]
 #else
-    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Basic_Authentication_Available))]
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
```

Comment:
The the whole point of this change, it enables Basic authentication by using a custom authenticator on the server side so you don't need to use real credentials to test it. As there's no such thing as ambient credentials with Basic authentication, there's no need to have a test variant which uses a real domain.